src/textarea/textarea.less

```diff
@@ -1,14 +1,76 @@
 .textarea-component {
     display: flex;
+
     textarea {
         height: auto;
+
         &:focus {
             outline: none;
             box-shadow: 0 0 1px fade(@info, 50%);
         }
+
         &[disabled] {
             background: @gray-light;
             cursor: not-allowed;
         }
     }
-}
+
+    .text-limit {
+        border-radius: 100%;
+        height: 12px;
+        width: 14px;
+        color: @gray-darker;
+        text-align: center;
+        margin: 5px auto auto;
+        letter-spacing: 0;
+        font-size: 10px;
+    }
+
+    circle {
+        stroke-width: 1;
+        fill: @white;
+    }
+
+    #gray {
+        stroke: @gray-stone;
+    }
+
+    .error-text {
+        stroke: @error;
+    }
+
+    .normal-text {
+        stroke: @gray-darker
+    }
+
+    .character-limited::-webkit-scrollbar {
+        width: 0.3em;
+        cursor: pointer;
+    }
+
+    .character-limited {
+        resize: none;
+    }
+
+    .character-limited::-webkit-scrollbar-track {
```

Comment:
Are you aware of that fact that this is vendor specific? https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar

Reply:
no i am not aware just saw this will fix it in firefox